backend/utils.py

```
@@ -183,29 +183,82 @@ def extract_diy_metadata(vision_data: dict, name: str, category: str):
 
 
 
+def _validate_sql_query(sql_query: str, username: str):
```

Ruff (0.14.4)

Missing return type annotation for private function `_validate_sql_query`

(ANN202)